.github/workflows/ci.yaml

```diff
@@ -185,7 +222,7 @@ jobs:
     strategy:
       max-parallel: 1
       matrix:
-        environment: ['production', 'staging', 'develop']
+        environment: ["production", "staging", "develop"]
```

Comment:
why changing to double `"`? I can see that there is lots of usage of single `'` and I strongly recommend that we use either one or the other, but not both

Reply:
It's replaced automatically by the linter. Some settings on my side must be off,

Reply:
using single quotes now